test/member.spec.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import {
+  Arc,
+  DevArcConfig as arcConfig,
+  DAO,
+  MemberData,
+  Member,
+  DAOData,
+} from "../src";
+import { render, screen } from "@testing-library/react";
+
+describe("Member component ", () => {
+  it("Shows member and dao address with inferred props", async () => {
+    const daoAddress = "0xe7a2c59e134ee81d4035ae6db2254f79308e334f";
+    const memberAddress = "0xe11ba2b4d45eaed5996cd0823791e0c93114882d";
+    const { container } = render(
+      <Arc config={arcConfig}>
+        <DAO address={daoAddress}>
+          <Member address={memberAddress}>
+            <DAO.Data>
+              <Member.Data>
+                {(dao: DAOData, member: MemberData) => (
```

Comment:
@cbrzn what do you think about this possible syntax:
```html
<Consume
  from={[DAO.Data, Member.Data, Member.Entity]}
  render={(dao: DAOData, member: MemberData, memberEntity: MemberEntity) => (
    <>
      <div>{member.address}</div>
      <div>{dao.id}</div>
    </>
  )}
/>
```

Reply:
That seems super straightforward, only think I'd change is that maybe just send the entity name, something like: `from={[DAO, Member]}` and the user can retrieve in the render method whatever he wants (Data and/or Entity from both)

Reply:
Sweet, I'll make an issue for this cosmetic change, which we can tackle in the future.